<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XSettings</class>
 <widget class="QDialog" name="XSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>XSettings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="XSettings.qrc">
    <normaloff>:/XSettings/settings1.ico</normaloff>
    <normalon>:/XSettings/settings1.ico</normalon>
    <disabledoff>:/XSettings/settings1.ico</disabledoff>
    <disabledon>:/XSettings/settings1.ico</disabledon>
    <activeoff>:/XSettings/settings1.ico</activeoff>
    <activeon>:/XSettings/settings1.ico</activeon>
    <selectedoff>:/XSettings/settings1.ico</selectedoff>
    <selectedon>:/XSettings/settings1.ico</selectedon>:/XSettings/settings1.ico</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignVCenter">
     <widget class="QLabel" name="label_header">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Leelawadee UI</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User settings</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_functionality">
      <property name="font">
       <font>
        <family>Leelawadee UI</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Functionality</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_sensitivity">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Joystick sensitivity (mouse sense):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="js_sense">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_triggerSense">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Trigger sensitivity (scroll sense):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="trigger_sense">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_maxReconnect">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Maximum reconnect time (in minutes):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="max_reconnects">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_administrative">
      <property name="font">
       <font>
        <family>Leelawadee UI</family>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Administrative</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="startWithSystem">
      <property name="text">
       <string>Start XWindows on system startup</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>419</y>
     <width>333</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="XSettings.qrc"/>
 </resources>
 <connections/>
</ui>
